src/test/resources/mocks/cop-daily-cause-list/copDailyCauseList.json

```diff
@@ -65,7 +63,7 @@
                 ],
                 "judiciary": [
                   {
-                    "johKnownAs": "Crown Judge",
+                    "johKnownAs": "Judge TestName Presiding",
                     "isPresiding": true
```

Comment:
COP list schema does not have isPresiding flag currently. Need to double check this. If so all COP test files should not have this field.